### ISSUE TITLE
Curve of train and val loss of DPRNN_TasNet

### ISSUE THREAD

**Issue by lijie03** [JusperLee/Dual-Path-RNN-Pytorch#4]
Hi Kai,

Could you pls. share the train and val loss curve of DPRNN TasNet? Just as that of ConvTasNet in your Result part. 

Thanks a lot!

**Comment by JusperLee** [comment]
Because the training in the middle was interrupted, the complete training curve cannot be obtained.
